src/pages/FirebaseTest.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import fb from '../lib/firebase';
+import firebase from 'firebase/app';
+
+export default function FirebaseTesting (props) {
```

Comment:
It looks like `props` is never been used. Do we need to keep it here?

Reply:
Removed in commit 1a220e5